ads/_config.js

```diff
@@ -51,6 +52,10 @@ export const adPreconnect = {
     'https://trc.taboola.com',
     'https://images.taboola.com',
   ],
+  criteo: [
+    'https://static.criteo.net',
```

Comment:
No need to repeat host from prefetching (we also preconnect in browser where we can't prefetch).